crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -895,7 +973,11 @@ impl ExecutionArgs {
         // default to false
         track_usage!(telemetry, self.framework_inference, |val: bool| !val);
 
-        track_usage!(telemetry, self.continue_execution, |val| val);
+        telemetry.track_arg_value(
```

Comment:
since we emit new values here (none/independent-tasks-only/all), should this be done under a different key? otherwise do we risk breaking consumers of this data who might expect the old values (set/default)?

Reply:
Just for to keeping data clean as we'll have events coming in from old/new versions we should make this a new key and keeping a proxy key.
We can change the old `track_usage!` call to:
```
track_usage!(telemetry, self.continue_execution, |val| matches!(val, ContinueMode::All | ContinueMode::IndependentTasksOnly));
```
It doesn't properly capture use of `--continue=none` vs not having `--continue` set, but I think that's fine. Merging these events is worth it for having defaults encoded in `clap`.